twoliter/embedded/Makefile.toml

```diff
@@ -97,7 +97,8 @@ BUILDSYS_JOBS = "8"
 CARGO_HOME = "${BUILDSYS_ROOT_DIR}/.cargo"
 # This needs to end with pkg/mod so that we can mount the parent of pkg/mod as GOPATH.
 GO_MOD_CACHE = "${BUILDSYS_ROOT_DIR}/.gomodcache/pkg/mod"
-GO_MODULES = "ecs-gpu-init host-ctr"
+# Dynamically load a list of go modules from ${BUILDSYS_SOURCE_DIR}
+GO_MODULES = { script = ['find ${BUILDSYS_SOURCES_DIR} -name go.mod -type f -printf "%h\n" | xargs -n1 basename'] }
```

Comment:
Do we want to do this with Rust prior to the calling of `cargo make` and pass the found go modules with `-e GO_MODULES`? I'm thinking now is as good a time as any to start moving logic away from Makefile.toml (or at least not adding more logic to it).